autoadmin.py

```diff
@@ -3,8 +3,11 @@
 
 
 # TODO Load from file
-ADMINS = {
-}
+try:
+    ADMINS = {admin_id for admin_id in open('ADMIN.txt', 'r').readlines()}
```

Comment:
1. You should use `with open`.
2. This isn't going to work (`admin_id = '123\n'`, which is `str` and not `int`).